### PR TITLE
blog post - prep 40 ppl for unconf

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

**Comment by stefaniebutland** [comment]
I'd like to number the questions in the "Questions included" section but using markdown like
1. text
2. text
the numbers don't appear when rendered

File: content/blog/2017-11-24-unconf-welcome.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+topicid:
+tags:
+- R
+- community
+- unconf
+- ropensci-team
+- welcome
+---
+
+I’ve raved about the value of [extending a personalized welcome](https://ropensci.org/blog/2017/07/18/value-of-welcome/) to new community members and I recently shared [six tips for running a successful hackathon-flavoured unconference](https://ropensci.org/blog/2017-11-17-unconf-sixtips). Building on these, I’d like to share the specific approach and (free!) tools I used to help prepare new rOpenSci community members to be productive at our [unconference](http://unconf17.ropensci.org/). My approach was inspired directly by my [AAAS Community Engagement Fellowship Program](https://blog.trelliscience.com/introducing-the-2017-community-engagement-fellows/) (AAAS-CEFP) training. Specifically, 1) one mentor said that the most successful conference they ever ran involved having one-to-one meetings with all participants prior to the event, and 2) prior to our in-person AAAS-CEFP training, we completed an intake questionnaire that forced us to consider things like “what do you hope to get out of this” and “what do you hope to contribute”.
```
Comment (sckott) [inline content/blog/2017-11-24-unconf-welcome.md]: this link format `https://ropensci.org/blog/2017-11-17-unconf-sixtips` is wrong, date with slashes separating numbers, not dashes

**Comment by sckott** [comment]
for the numbering, try indenting four spaces the text between the numbered parts https://stackoverflow.com/a/18089124/1091766

File: content/blog/2017-11-24-unconf-welcome.md (hunk 1 above)
Comment (stefaniebutland) [reply content/blog/2017-11-24-unconf-welcome.md]: fixed

**Comment by stefaniebutland** [comment]
need to add some headings

**Comment by stefaniebutland** [comment]
> for the numbering, try indenting four spaces the text between the numbered parts https://stackoverflow.com/a/18089124/1091766
didn't have any effect

**Comment by stefaniebutland** [comment]
@jeroen do you have an idea how to fix this?
I'd like to number the questions in the "Questions included" section but using markdown like
1. text
  - example
2. text
  - example
but the numbers don't appear when rendered. 
I also tried a pure ordered list without the bullet items below, but still, the numbers didn't show in the preview

**Comment by jeroen** [comment]
@marulango I think the `list-style-type` for `ol` are hidden? Can we enable those somehow?

**Comment by marulango** [comment]
I just pushed https://github.com/ropensci/roweb2/commit/948b64ff51db08835c9bc5f1327447545839875b and that fixes the list-style-type thing, wasn't defined after I overrided the browsers default spacing

**Comment by jeroen** [comment]
@stefaniebutland if you merge the `master` branch into this branch, the numbered lists should now be visitble!

**Comment by stefaniebutland** [comment]
@marulango thanks for this
>I just pushed 948b64f and that fixes the list-style-type thing, wasn't defined after I overrided the browsers default spacing

Can you please have a look at this formatting problem with lists? When a numbered line wraps to second line, the number is aligned with second line. Same happens with bullets in this post: https://ropensci.org/blog/2017/10/06/sholler-plan/

**Pull request merged**